minorminer/utils/raster_embedding.py

```diff
@@ -117,27 +117,42 @@ def find_multiple_embeddings(S, T, timeout=10, max_num_emb=float('inf')):
             Defaults to 10.
         max_num_emb (int, optional): Maximum number of embeddings to find.
             Defaults to inf (unbounded).
+        inplace (bool, optional): Specifies whether to remove visited nodes 
+            from the target graph. Defaults to False, which retains nodes, 
+            suitable for single embeddings. Set to True if searching for multiple embeddings.
+        skip_filter (bool, optional): Specifies whether to skip the subgraph 
+            lower bound filter. Defaults to True, meaning the filter is skipped.
     Returns:
         list: A list of disjoint embeddings. Each embedding defines a 1:1 map
             from the source to the target in the form of a dictionary with no
             reusue of target variables.
     """
-    _T = T.copy()
     embs = []
-    while True and len(embs) < max_num_emb:
-        # A potential feature enhancement would be to allow different embedding
-        # heuristics here, including those that are not 1:1
-        
-        if subgraph_embedding_feasibility_filter(S, T):
+    if max_num_emb != 1:
+        inplace = False
+    if inplace:
+        _T = T
+        if skip_filter or subgraph_embedding_feasibility_filter(S, T):
```

Comment:
If inplace = True it looks like the only possibility is to have max_num_emb=1, which is not what we want. I think that lines 135-138 need to be deleted, and then lines 141-155 unindented? This makes clearest use of the argument and best reuse of the code I think

Reply:
There is a problem. If `inplace = True` and we set `_T = T ` we cannot perform the remove operation beacuse the function will not let us modify graph T.